src/Felix/Instances/Setoid/Raw.agda

```diff
@@ -0,0 +1,153 @@
+{-# OPTIONS --safe --without-K  #-}
+
+module Felix.Instances.Setoid.Raw where
+
+open import Data.Product using (_,_; <_,_>; proj₁; proj₂; uncurry′; ∃₂)
+open import Data.Product.Relation.Binary.Pointwise.NonDependent using (_×ₛ_)
+open import Data.Sum using ([_,_]; inj₁; inj₂)
+open import Data.Sum.Relation.Binary.Pointwise using (inj₁; inj₂)
+open import Data.Unit.Polymorphic using (tt) renaming (⊤ to ⊤′)
+open import Function using (Congruent; Func; _∘′_; flip′) renaming (_∘_ to _∘ᵈ_)
+open import Function.Construct.Composition as Comp
+open import Function.Construct.Constant as Const
```

Comment:
Where is this module? I'm not finding it on my machine. Maybe I'm out of sync with the released 2.0.

Reply:
Yes, [it is in 2.0](https://agda.github.io/agda-stdlib/v2.0/Function.Construct.Constant.html), it's most likely that your one is out of date